api/.env.sample

```diff
@@ -27,4 +27,6 @@ TWILIO_ACCOUNT_SID=
 TWILIO_AUTH_TOKEN=
 TWILIO_FROM_NUMBER=
 
+CREATE_ADMIN_USER=false
```

Comment:
Why is this needed locally? Won't the migration just gracefully exit if the env var is not set?

https://github.com/govau/notify/blob/3e886de5e8943418602ef751a4c2dc286bc5a8ff/api/migrations/versions/0200_create_admin_user.py#L24

Reply:
Actually you're right...too early in the week